drivers/net/ethernet/renesas/rswitch_tc_u32.c

```diff
@@ -5,10 +5,6 @@
  * Copyright (C) 2022 EPAM Systems
  */
 
-#include <net/tc_act/tc_mirred.h>
```

Comment:
Should it be in this PR?

Reply:
The change has been removed from this PR. Thank you for the close review:)